services/rest/websocket/connection/table.go

```diff
@@ -44,11 +44,26 @@ func (t *Table) Store(c identification.Identifier) error {
 	if _, ok := t.connMap[c]; ok {
 		return errConnDuplicated
 	}
-	t.connMap[c] = struct{}{}
+	t.connMap[c] = make(map[string]struct{})
 	t.counter[c.Group] = t.counter[c.Group] + 1
 	return nil
 }
 
+func (t *Table) StoreEvent(c identification.Identifier, eventId string) {
```

Comment:
Can we call this batchId as that's what we store.

Reply:
Okay, I will update to `requestId` or `reqGuid`

Reply:
done.